Sources/Commands/SwiftTool.swift

```diff
@@ -374,6 +382,16 @@ public class SwiftTool<Options: ToolOptions> {
                 usage: "Link Swift stdlib statically"),
             to: { $0.shouldLinkStaticSwiftStdlib = $1 })
 
+        binder.bind(
+            option: parser.add(option: "--enable-build-manifest-caching", kind: Bool.self,
```

Comment:
This is stale, you need to rebase I think

Reply:
Done